Resources/Prototypes/Roles/Jobs/Cargo/cargo_technician.yml

```diff
@@ -3,7 +3,7 @@
   name: job-name-cargotech
   description: job-description-cargotech
   playTimeTracker: JobCargoTechnician
-  antagAdvantage: 2 # DeltaV - Reduced TC: External Access
+  antagAdvantage: 0 # DeltaV - Revert TC penalty. 
```

Comment:
if this is the same as Wizden then this comment should be removed

Reply:
I think I fixed it! Thanks :)